src/main/java/seedu/address/ui/FoodCard.java

```diff
@@ -44,7 +44,7 @@ public FoodCard(Food food, int displayedIndex) {
         this.food = food;
         id.setText(displayedIndex + ". ");
         name.setText(food.getName().fullName);
-        phone.setText(food.getPhone().value);
+        price.setText(food.getPrice().value);
```

Comment:
Should add the dollar sign at the start:
price.setText("$" + food.getPrice().value);

Reply:
Have made the changes in [this commit](https://github.com/AY1920S1-CS2103T-F13-2/main/pull/75/commits/c56568a6be094653d109905bb600107cf31a1ac5), thanks!